src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/Marshalling/GeneratedComClassAttribute.cs

```diff
@@ -3,6 +3,9 @@
 
 namespace System.Runtime.InteropServices.Marshalling
 {
+    /// <summary>
+    /// Specifies that the attributed type will be exposed to COM through source-generated COM and that the source generator should generate code for it.
+    /// </summary>
```

Comment:
I think it would be helpful to add a remark that the class needs to implement a source generated interface.